src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/RazorDocumentMappingServiceTest.cs

```diff
@@ -1035,7 +1035,7 @@ private static RazorCodeDocument CreateCodeDocumentWithCSharpProjection(string r
         var csharpDocument = new RazorCSharpDocument(
             codeDocument,
             projectedCSharpSource,
-            RazorCodeGenerationOptions.CreateDefault(),
+            RazorCodeGenerationOptions.            Default,
```

Comment:
What happened here!? 😛

Reply:
Ugh. I missed that one. There were a few of these after applying the "method to property" refactoring. Thanks for noticing! I'll fix that up.